### PR TITLE
Fix number attribute error

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes: https://github.com/thoth-station/kebechet/issues/281

### CODE REVIEW

**Comment by goern** [comment]
recheck

**Comment by bissenbay** [comment]
> recheck

I'm just preventing the case when `issue` can be equal to `None`.

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -454,8 +454,9 @@ def _relock_all(self, exc: PipenvError, labels: list) -> None:
         commit_msg = "Automatic dependency re-locking"
         branch_name = "kebechet-dependency-relock"
         self._git_push(":pushpin: " + commit_msg, branch_name, ['Pipfile.lock'])
-        pr_id = self.sm.open_merge_request(commit_msg, branch_name, f"Fixes: #{issue.number}", labels)
-        _LOGGER.info(f"Issued automatic dependency re-locking in PR #{pr_id} to fix issue #{issue.number}")
+        if issue is not None:
+            pr_id = self.sm.open_merge_request(commit_msg, branch_name, f"Fixes: #{issue.number}", labels)
+            _LOGGER.info(f"Issued automatic dependency re-locking in PR #{pr_id} to fix issue #{issue.number}")
```
Comment (fridex) [inline kebechet/managers/update/update.py]: Not sure if this is the correct way of fixing this. This way we will not open the desired PR. What about always returning issue from the following code?

https://github.com/thoth-station/kebechet/blob/bcf908474acf25cdcd563fb85521776b608778ad/kebechet/source_management.py#L73-L95

Note sure about the impact of this change (usage of `open_issue_if_not_exist` elsewhere in kebechet).

Comment (bissenbay) [reply kebechet/managers/update/update.py]: @fridex thanks, done!

**Comment by bissenbay** [comment]
@fridex Please TAL?

**Comment by bissenbay** [comment]
How are you, @fridex !

**Pull request merged**